src/pages/index.js

```diff
@@ -55,33 +137,62 @@ const cardSection = new Section(
   ".cards__list"
 );
 
-cardSection.renderItems();
-
-// Create PopupWithForm for profile edit
 const profilePopup = new PopupWithForm("#profile-edit-modal", (formData) => {
-  userInfo.setUserInfo({
-    name: formData.title,
-    job: formData.description,
-  });
-  profilePopup.close();
+  profilePopup.renderLoading(true);
+  api
+    .updateUserProfile({ name: formData.title, about: formData.description })
+    .then((updatedUser) => {
+      userInfo.setUserInfo({
+        name: updatedUser.name,
+        job: updatedUser.about,
+      });
+      profilePopup.close();
+    })
+    .catch((err) => console.error(err))
+    .finally(() => {
+      profilePopup.renderLoading(false);
+    });
 });
 profilePopup.setEventListeners();
 
-// Create PopupWithForm for card addition
 const addCardPopup = new PopupWithForm("#profile-card-modal", (formData) => {
+  addCardPopup.renderLoading(true);
   const cardData = { name: formData["title"], link: formData["url"] };
-  const newCardElement = createCard(cardData);
-  cardSection.addItem(newCardElement);
-  addCardPopup.close();
-  addCardPopup.resetForm();
-  cardFormValidator.disableButton();
+  api
+    .addCard(cardData)
+    .then((newCard) => {
+      const newCardElement = createCard({
+        ...newCard,
+        userId: userInfo.getUserInfo()._id,
+      });
+      cardSection.addItem(newCardElement);
+      addCardPopup.close();
```

Comment:
You had `addCardPopup.resetForm();` in the 8th sprint. Please, bring it back to clear the form here in `then`